src/@next/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Typography } from '../Typography';
+import { Neutral } from '../utilities/colors';
+import { AvatarStyle } from './AvatarStyle';
+
+const avatarBackgroundColorVariant = [
+  'supplementary',
+  'warning',
+  'danger',
+  'success',
+  'outstanding',
+] as const;
+
+const avatarSizeVariant = ['large', 'medium'] as const;
+
+export type AvatarBackgroundColorVariant =
+  typeof avatarBackgroundColorVariant[number];
+
+export type AvatarSizeVariant = typeof avatarSizeVariant[number];
+
+export interface AvatarProps extends React.HTMLAttributes<HTMLDivElement> {
+  variant?: AvatarBackgroundColorVariant;
+  size?: AvatarSizeVariant;
+  initials?: string;
+}
+
+export const Avatar = ({
+  variant = 'supplementary',
+  size = 'medium',
+  initials,
+  ...props
+}: AvatarProps) => {
+  return (
+    <AvatarStyle variant={variant} size={size ?? 'large'} {...props}>
```

Comment:
```suggestion
    <AvatarStyle variant={variant} size={size} {...props}>
```
we should be able to remove this because of the default value above.

Reply:
removed